src/configdialog.ui

```diff
@@ -4046,7 +4046,23 @@ them here.</string>
                     </item>
                    </widget>
                   </item>
-                  <item row="1" column="0">
+                  <item row="1" column="1">
+                   <widget class="QCheckBox" name="checkBoxAutoPreviewCmd">
+                    <property name="text">
+                     <string notr="true">Prefer Preview with Build Compiler</string>
+                    </property>
+                    <property name="toolTip">
+                     <string>The option applies when the build compiler is pdflatex, lualatex, xelatex, or latex.</string>
```

Comment:
Would "is one of ..." be better? I'm not native speaker.
```suggestion
                     <string>The option applies when the build compiler is one of pdflatex, lualatex, xelatex, or latex.</string>
```

Reply:
I'll wait for suggestions from sunderme